### ISSUE TITLE
Not including generated files busts the node module

### ISSUE THREAD

**Issue by nwinter** [vickychijwani/closer.js#13]
Checking out what gets included in npm:

```
node_modules
demo
package.json
spec
src
.bowerrc
.npmignore
.travis.yml
Gruntfile.coffee
LICENSE
README.md
repl
```

For some reason, it no longer has `lib` in there, so the entry point in package.json (`"main": "lib/src/index.js"`) doesn't work. Probably happened when changing to only include the compiled sources in the `gh-pages` branch.

Should be able to get the compiled code but not source in npm and the source but not compiled code in GitHub using .gitignore and .npmignore, right?

**Comment by vickychijwani** [comment]
That's strange. The `.npmignore` doesn't have `lib` in it:

```
.idea
node_modules
/npm-debug.log
src/
spec/
demo/
.bowerrc
.gitignore
.git
Gruntfile.coffee
```

Looking into it now.

**Comment by vickychijwani** [comment]
Don't know what went wrong, but I've fixed it and published v0.2.2 to npm. I've also changed it to whitelist files for the npm package rather than blacklisting (commit 6b5ebde), that should make the package much more reliable in the future.

**Comment by nwinter** [comment]
Thanks, that did the trick! (After I realized I need to npm cache clean to get the update properly.)
